specification/network/resource-manager/readme.md

```diff
@@ -940,6 +938,12 @@ directive:
   - suppress: RequiredPropertiesMissingInResourceModel
     from: applicationSecurityGroup.json
     reason: name, id and type properties are inherited from the upper level
+  - suppress: RequiredPropertiesMissingInResourceModel
+    from: autoApprovedPrivateLinkServices.json
```

Comment:
Since there is no `autoApprovedPrivateLinkServices.json` file, you don't need this exception.
Same for `availablePrivateEndpointTypes.json` and `checkPrivateLinkServiceVisibility.json`

Reply:
Fixed. Thanks.